doc/source/data/dataset.rst

```diff
@@ -145,6 +145,10 @@ Datasource Compatibility Matrices
 Creating Datasets
 -----------------
 
+.. tip
```

Comment:
This isn't correct rst, and doesn't show up in the compiled output. You can see the CI preview build of the readthedocs by clicking within a few minutes of push.

Reply:
I think this is all that's needed.
```suggestion
.. tip::
```